environment.yml

```diff
@@ -16,3 +16,5 @@ dependencies:
     - tqdm
     - joblib
     - click
+    - git+https://github.com/cortex-lab/phy.git@dev
+    - git+https://github.com/cortex-lab/phy-contrib.git@dev
```

Comment:
to replace with `git+https://github.com/cortex-lab/phylib.git`
phy-contrib is deprecated in phy2

Reply:
Done

Reply:
thanks but the first line (phy) was good!